tests/hash/main.cpp

```diff
@@ -144,6 +152,55 @@ int main(int argc, char *argv[]) {
     }
   }
   input.open(argv[2], ios_base::in);
+
+  if(f == hash_blake2b) {
```

Comment:
The indentation doesn't match with other parts of the code. You sometimes use 2 spaces and sometimes 4.

Reply:
Editor issue. Will fix this. Have to set up Monero configuration for my editor.